Example/DApp/Modules/Sign/SessionAccount/SessionAccountView.swift

```diff
@@ -179,7 +189,18 @@ struct SessionAccountView: View {
             }
         }
     }
-    
+
+    private var loadingView: some View {
+        VStack {
+            ProgressView()
+                .progressViewStyle(CircularProgressViewStyle(tint: .black))
+                .scaleEffect(1.5)
+            Text("Request sent, waiting for response")
+                .foregroundColor(.white)
+                .padding(.top, 20)
+        }
+    }
+
     private func responseView(response: Response) -> some View {
```

Comment:
you can pass lastRequest as param to avoid force unwrap 
```
if let lastRequest = presenter.lastRequest {
responseView(response: response, lastRequest: lastRequest)
}
```